k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -6,6 +6,9 @@ storage_class='local-path'
 ##TODO Extract from solution.yaml ?
 serviceAccountName=cortx-sa
 
+SERVER_POD_DEPLOYMENT=true
```

Comment:
Please add comment about meaning of these.e.g. if we need Data Only Pod Deployment what should be these values be set to ?

Reply:
fixed with deployment mode approach